draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -280,9 +280,9 @@ First, the CA issues a certificate to the authenticating party:
 Next, tree heads flow to the relying party, after being durably and consistently logged. This occurs periodically in the background, unconnected from uses of individual certificates:
 
 {:start="4"}
-4. The transparency service periodically downloads the abridged assertions, recreates the Merkle Tree, and validates the window signature. It mirrors them for monitors to observe. See {{transparency-service}}.
+4. Mirrors periodically download the abridged assertions, recreate the Merkle Tree, and validate the window signature. They mirror the contents for monitors to observe. See {{transparency-ecosystem}}.
 
-5. The relying party periodically fetches the latest validity window from the transparency service. This validity window will contain the new tree head.
+5. The relying party periodically fetches the latest validity window from mirrors that it trusts. This validity window will contain the new tree head.
```

Comment:
I think we're missing the distinction between the RP and the RP's out-of-band update service here. This is important because of the privacy implications of a browser reaching out directly to mirrors, for example. Maybe we need separate boxes for those two entities to make things crystal clear?
```suggestion
5. The relying party's update service periodically fetches the latest validity window from mirrors that it trusts. This validity window will contain the new tree head.
```

Reply:
The change to de-emphasize the RP directly contacting the mirror looks good!